samples/grids/data-grid/load-save-layout/src/index.tsx

```diff
@@ -0,0 +1,378 @@
+
+                    {/* <IgrTextColumn field="Country" width="*>140" horizontalAlignment="center"/> */}
+                    <IgrTextColumn field="Income" width="*>130" horizontalAlignment="center"/>
+                    <IgrTextColumn field="Age" width="*>110" horizontalAlignment="center"/>
+               </IgrDataGrid>
+            </div>
+        );
+    }
+
+    public onGridRef(grid: IgrDataGrid) {
+        if (!grid) { return; }
+
+        this.grid = grid;
+        this.grid.actualDataSource.isSectionExpandedDefault = true;
+        
+        if (this.toolbar !== null) {
+            this.toolbar.targetGrid = this.grid;
+        }
+    }
+
+    public onToolbarRef(toolbar: IgrDataGridToolbar) {
+          this.toolbar = toolbar;
+          if (this.toolbar !== null) {
+              this.toolbar.targetGrid = this.grid;
+          }
+    }
+
+    public componentDidMount() {
+
+        const peopleGroup = new IgrColumnGroupDescription();
+        peopleGroup.field = "Country";
+        peopleGroup.displayName = "Country";
+        this.grid.groupDescriptions.add(peopleGroup);
+
+        const incomeGroup = new IgrColumnGroupDescription();
+        incomeGroup.field = "Income";
+        incomeGroup.displayName = "Income";
+        this.grid.groupDescriptions.add(incomeGroup);
+
+        const peopleCount = new IgrColumnSummaryDescription();
+        peopleCount.field = "Photo";
+        peopleCount.operand = SummaryOperand.Count;
+        this.grid.summaryDescriptions.add(peopleCount);
+
+        const sales = new IgrColumnSummaryDescription();
+        sales.field = "Sales";
+        sales.operand = SummaryOperand.Max;
+        sales.formatOverride = new Intl.NumberFormat('en-EN', { style: 'currency', currency: 'USD', minimumFractionDigits: 0, maximumFractionDigits: 0 });
+        this.grid.summaryDescriptions.add(sales);
+
+        const salary = new IgrColumnSummaryDescription();
+        salary.field = "Salary";
+        salary.operand = SummaryOperand.Average;
+        salary.formatOverride = new Intl.NumberFormat('en-EN', { style: 'currency', currency: 'USD', minimumFractionDigits: 0, maximumFractionDigits: 0 });
+        this.grid.summaryDescriptions.add(salary);
+    }
+
+    public getProductivityChart(props: IIgrCellTemplateProps) {
+        const info = props.dataContext as IgrTemplateCellInfo;
+        return (
+            <div className="container">
+               <IgrSparkline
+                   height="30px" width="100%"
+                   displayType="Column"
+                   dataSource={info.rowItem.Productivity}
+                   valueMemberPath="Value"
+                   labelMemberPath="Week"
+                   lineThickness={2}
+                   brush="rgb(21, 190, 6)"
+                   negativeBrush="rgb(211, 17, 3)" />
+            </div>
+        );
+    }
+
+    public onAddressCellUpdating(s: IgrTemplateColumn, e: IgrTemplateCellUpdatingEventArgs) {
```

Comment:
remove this function to make this sample less complex

Reply:
resolved

https://github.com/IgniteUI/igniteui-react-examples/pull/101/commits/af54a47b9c02875025df7f83cf81dc8e6813eb85